runtime/queries/css/highlights.scm

```diff
@@ -57,7 +57,7 @@
  (attribute_name)
  ] @property
```

Comment:
These highlights should be fixed if that's not out of scope.
I'll take another look later to figure out what they should be.

Reply:
I'd vote for keeping all `@property` stuffs for another PR, or to fix them all in this one. Might be hard as it's not just a rename though.

Reply:
Yes, I think we should keep fixup (changing as opposed to renaming captures) to separate PR(s); that'll make it easier to get the language maintainers involved and can happen in parallel. The more that needs to be done in this PR, the longer everything gets blocked.

Remember that the main point of this PR is to change the _allowed_ captures in CONTRIBUTING.md; changing the queries is just a necessary consequence thereof.